mobile/src/modules/auth/components/AccountsListHeader.tsx

```diff
@@ -43,7 +47,9 @@ export class AccountsListHeader extends React.PureComponent<Props> {
         </Text>
         {priceInBTC ? (
           <Text color={Colors.WHITE} bebasFont>
-            {i18n.t(core.currency.BTC.value, { value: amountToString(totalBalanceBTC) })}
+            {priceApi && priceApi.selectedCurrency === 'USD' ?
+              i18n.t(core.currency.USD.value, { value: totalBalanceUSD.toFixed(2) }) :
+              i18n.t(core.currency.BTC.value, { value: amountToString(totalBalanceBTC) })}
```

Comment:
Prop changes trigger `render` here just fine, just not in the AccountsListItem.tsx component.